etc/databases/world/updates/updates.sql

```diff
@@ -30510,5 +30510,76 @@ begin not atomic
         insert into applied_updates values ('080320231');
     end if;
 
+    -- 09/03/2023 1
+    if (select count(*) from applied_updates where id = '090320231') = 0 then
+        -- Set "Enchanter" subname for male Enchanting Vendors.
+        update creature_template set subname = 'Enchanter' where entry in(4228, 4617, 5758);
+        -- And "Enchantress" for females.
+        update creature_template set subname = 'Enchantress' where entry in(1318, 3012, 3346, 4228, 5158, 5757);
```

Comment:
Just noticed `4228 | Vaean` appears in both `Enchanter` and `Enchantress` queries. I assume he needs to be in the first query only.

Reply:
Fixed